src/app/login-register/service/user-register.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {FormControl} from "@angular/forms";
+import {User} from "../../classes/user";
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
  providedIn: "root"
```